test/python/test_coupling.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+
+# Copyright 2018, IBM.
+#
+# This source code is licensed under the Apache License, Version 2.0 found in
+# the LICENSE.txt file in the root directory of this source tree.
+
+# pylint: disable=invalid-name
+
+"""Coupling Graph Test: Directed graph object for representing coupling between qubits."""
+
+import unittest
+from qiskit.mapper._coupling import Coupling
+from qiskit.mapper._couplingerror import CouplingError
+from .common import QiskitTestCase
+
+
+class TestCoupling(QiskitTestCase):
+    """QISKit Coupling Graph Tests."""
+
+    def test_distance_error(self):
```

Comment:
What about moving this test (or the testcase) to `test_mapper.py`? Since the coupling functionality is actually inside the `qiskit.mapper` module, it seems that it might conceptually fit there (and have the nice side effect of having another new file with just a single testcase with a single test).

Reply:
The coupling graph is a structure that can be tested by itself, without a mapper. Additionally the code in `qiskit/mapper/_coupling.py` very poorly tested. I think `test/python/test_coupling.py` should be kept separately and need to be extended. Filing #770.